src/ui/simulator/toolbox/components/datagrid/component.cpp

```diff
@@ -663,7 +656,6 @@ void InternalState::onMapLayerRemoved(const wxString* text)
     // event
     if (pLayerFilter)
     {
```

Comment:
you removed unused **hzCombo** local variable, please remove comments referencing this variable.

Reply:
done